src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -232,7 +244,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.toggleEditMode(this.isActive());
 
 		if (this.output && this.output.nativeElement) {
-			(<HTMLElement>this.output.nativeElement).scrollTo({ behavior: 'smooth' });
+			(<HTMLElement>this.output.nativeElement).scrollIntoView({ behavior: 'smooth' });
```

Comment:
Is this fixing an issue or something?

Reply:
Not aware of any issue but scrollTo didn't work when I was debugging with my changes so I changed this to scrollIntoView which is being used on editors in vscode.